### PR TITLE
Better Sample App

### PR DESCRIPTION
**Pull request by sahil-shubham**
## What's in this PR? 
- improved tic tac toe app with styled components and added a restart game button

## TODO
- [x] ~add an option in sample app creation script to make an empty sample app rather than using tic tac toe~ made a simpler app

### CODE REVIEW

**Comment by hargup** [comment]
LGTM

File: src/apps/TicTacToe.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  return <Board />;
+};
+
+const shouldRunMyApp = async ({ query }) => {
+  const triggerQueries = ["tic tac toe"];
+  for (const triggerQuery of triggerQueries) {
+    if (query.toLowerCase() === triggerQuery) {
+      return { query };
+    }
+  }
```
Comment (OrkoHunter) [inline src/apps/TicTacToe.js:137]: a substring match might be helpful to serve queries like `play tic tac toe` :)

File: src/apps/TicTacToe.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    if (squares[a] && squares[a] === squares[b] && squares[a] === squares[c]) {
+      return squares[a];
+    }
+  }
+  return null;
+}
+
+const MyAppComponent = (props) => {
+  return <Board />;
+};
```
Comment (OrkoHunter) [inline src/apps/TicTacToe.js:129]: Looks like this component is useless 😅

File: src/apps/TicTacToe.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    padding: 0;
+    text-align: center;
+    width: 34px;
+  `;
+
+  return <StyledSquare onClick={props.onClick}>{props.value}</StyledSquare>;
+}
+
+const Board = () => {
+  const [squares, setSquares] = useState(Array(9).fill(null));
```
Comment (OrkoHunter) [inline src/apps/TicTacToe.js:47]: TIL `Array(N).fill()` 🤯 

Was looking for the Pythonic syntax in JS the other day - `[0]*9`

File: src/apps/TicTacToe.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { InstantApp } from "./types";
```
Comment (OrkoHunter) [inline src/apps/TicTacToe.js:3]: Is this being used, since the file is `.js`?

**Review by OrkoHunter — APPROVED** [review]

File: src/apps/TicTacToe.js (hunk 4 above)
Comment (sahil-shubham) [reply src/apps/TicTacToe.js:3]: Nice catch, I was on my way to write to change it into typescript but left it for later. Prettier should have removed it.

**Pull request merged**